core/src/main/scala/kafka/tools/TestLogCompactionTool.scala

```diff
@@ -46,64 +51,60 @@ import scala.collection.JavaConverters._
  * Then we compare the final message in both logs for each key. If this final message is not the same for all keys we
  * print an error and exit with exit code 1, otherwise we print the size reduction and exit with exit code 0.
  */
-object TestLogCleaning {
+object TestLogCompactionTool {
```

Comment:
This should remain in the test directory right? It's similar to `MiniKdc`.

Reply:
It is similar to ReplicaVerificationTool. So i moved to src directory.  This will be useful, if we want to test manually with binary distribution.

Also, adding testlibs to minikdc service classpath handled separately. not sure we want do same here.
https://github.com/apache/kafka/blob/trunk/tests/kafkatest/services/security/minikdc.py#L110MiniKdc

Reply:
A KIP is needed for adding new user visible tools. Personally, I don't see much value in testing the binary distribution manually.

Reply:
Ok..  moved tool class to test directory and retained the original name.